### PR TITLE
feat(config): support JSON files

### PR DESCRIPTION
**Pull request by ronaudinho**
https://github.com/aquasecurity/trivy/issues/834

### CODE REVIEW

File: analyzer/config/json/json.go [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+	for _, required := range requiredExts {
+		if ext == required {
+			return true
+		}
+	}
+	return false
+}
+
+func (a jsonConfigAnalyzer) Type() analyzer.Type {
+	return analyzer.TypeYaml
```
Comment (knqyf263) [inline analyzer/config/json/json.go:54]:
```suggestion
	return analyzer.TypeJSON
```

Comment (knqyf263) [reply analyzer/config/json/json.go:54]: I didn't add any test of `Type()` in the YAML analyzer, but we might need to add a pretty simple test.

**Review by knqyf263 — CHANGES_REQUESTED** [review]

File: analyzer/config/json/json.go (hunk 1 above)
Comment (ronaudinho) [reply analyzer/config/json/json.go]: added test for `Type()` here. I just realized that `Version()` does not have test too, would it be necessary?

**Review by knqyf263 — APPROVED** [review]
:shipit:

**Pull request merged**